ipaserver/install/dnskeysyncinstance.py

```diff
@@ -164,6 +165,87 @@ def __setup_dnssec_containers(self):
 
         self._ldap_mod("dnssec.ldif", {'SUFFIX': self.suffix, })
 
+    def __are_named_options_configured(self, options):
```

Comment:
Please no more double under prefix. They are annoying.

```suggestion
    def _are_named_options_configured(self, options):
```

Reply:
I'm trying to follow the existing code style. Of course, I don't plan to annoy you.

Reply:
Fixed.